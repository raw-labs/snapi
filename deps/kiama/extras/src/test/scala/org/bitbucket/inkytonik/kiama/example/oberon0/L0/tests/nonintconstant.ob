MODULE nonintconstant;

CONST
    aconst1 = 125;
    aconst2 = TRUE;
    aconst3 = TRUE OR (aconst1 = 100);

END nonintconstant.
