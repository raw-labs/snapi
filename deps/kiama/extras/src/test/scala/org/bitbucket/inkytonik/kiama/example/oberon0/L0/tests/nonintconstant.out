* _pp.ob
MODULE nonintconstant;

    CONST
        aconst1 = 125;
        aconst2 = TRUE;
        aconst3 = TRUE OR (aconst1 = 100);

END nonintconstant.
[3]* errors
[3]src/test/scala/org/bitbucket/inkytonik/kiama/example/oberon0/L0/tests/nonintconstant.ob:5:15:error: got BOOLEAN, but expected INTEGER
[3]    aconst2 = TRUE;
[3]              ^
[3]src/test/scala/org/bitbucket/inkytonik/kiama/example/oberon0/L0/tests/nonintconstant.ob:6:15:error: got BOOLEAN, but expected INTEGER
[3]    aconst3 = TRUE OR (aconst1 = 100);
[3]              ^
